target/scripts/helpers/error.sh

```diff
@@ -77,22 +76,14 @@ function _dms_panic__general       { dms_panic 'general'       "${1:-}" "${2:-}"
 
 # Call this method when you want to panic (i.e. emit an 'ERROR' log, and exit uncleanly).
 # `dms_panic` methods should be preferred if your failure type is supported.
+trap "exit 1" SIGUSR1
+SCRIPT_PID="$$"
```

Comment:
To keep up with our conventions, this should be

```suggestion
SCRIPT_PID=${$}
```

which I agree looks interesting 😆